test/e2e/validation_test.go

```diff
@@ -254,3 +261,41 @@ func filterSSMatrix(mat *types.ComMatrix) (*types.ComMatrix, error) {
 
 	return &types.ComMatrix{Matrix: res}, nil
 }
+
+func extractNotUsedAndMissingEPS(csvContent string) (*types.ComMatrix, *types.ComMatrix, error) {
+	plusLinesCSV := filterCSVPlusMinusLines(csvContent, "+")
+	minusLines := filterCSVPlusMinusLines(csvContent, "-")
+
+	notUsedEPSMat, err := types.ParseToComDetailsList([]byte(plusLinesCSV), types.FormatCSV)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	missingEPSMat, err := types.ParseToComDetailsList([]byte(minusLines), types.FormatCSV)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return &types.ComMatrix{Matrix: notUsedEPSMat}, &types.ComMatrix{Matrix: missingEPSMat}, nil
+}
+
+func filterCSVPlusMinusLines(csvContent string, prefix string) string {
```

Comment:
A better name could be `filterCsvLinesByPrefix` as this code is generic.

Reply:
Seems better